src_kt/javatests/arcs/util/BUILD

```diff
@@ -33,6 +33,8 @@ arcs_kt_jvm_test_suite(
     name = "util_performance",
     srcs = PERFORMANCE_SRCS,
     package = "arcs.util",
+    # Disabled due to unreliable performance characteristics.
+    tags = ["manual"],
```

Comment:
These tags are the tests that are disabled? Or are we enabling them now and they were previously broken? Comment and code either mismatch or are just surprising to me.

Reply:
The "manual" tag means it won't be included when you try to test all, e.g. using `bazel test //src_kt/...`

This test still works. It's just a bit slow, so we don't want it on Travis. Previously it was disabled via @Ignore, but now it's not disabled, it's just a manual test. You can still run it if you want.